cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/permission/IPermissionManagement.java

```diff
@@ -176,6 +176,10 @@ default IPermissionGroup addGroup(String role, int potency) {
             }
         }
 
+        if (permissionGroups.isEmpty()) {
+            permissionGroups.add(this.getDefaultPermissionGroup());
+        }
```

Comment:
Is this being saved?

Reply:
No

Reply:
PermissionUser#groups is the list that is saved